app/models/miq_action.rb

```diff
@@ -799,7 +806,7 @@ def action_cancel_task(action, rec, inputs)
     end
 
     MiqPolicy.logger.info("MIQ(action_cancel_task): Now executing Cancel of task [#{source_event.event_type}] on VM [#{source_event.vm_name}]")
-    ems = ExtManagementSystem.find_by(:id => source_event.ems_id)
+    ems = ExtManagementSystem.find_by_id(source_event.ems_id)
```

Comment:
@gmcculloug 
since i need the miq_action changes i might have picked up extra changes from master. in only need the line 228 update